File: stronghold/stronghold_cpp/dna.cpp
// /usr/bin/g++ - you don't need a shebang line for cpp
#include <iostream>
#include <fstream> //file i/o
using namespace std;

int main(){
	cout<<"hello world";
	fstream x;
	x.open("data/rosalind_dna.txt",ios::in);
	return 0;
}

File: pyrosalind/dna.py
#!/usr/bin/python3

file=open("data/rosalind_dna.txt",'r')
#x='AGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC'
x=str(file.read())
a,t,g,c=0,0,0,0
for i in x:
    if i=='A':
        a+=1
    if i=='T':
        t+=1
    if i=='G':
        g+=1
    if i=='C':
        c+=1

print(a,c,g,t,sep=" ")
